### DISCUSSION TITLE
Is it possible to not render the right side table of contents for specific pages?

### DISCUSSION THREAD

**Discussion by linusang-spark** [vuejs/vitepress#4366] [Q&A] [answered]
question as per title

**Comment by brc-dd** [comment, marked as answer]
add `outline: false` to frontmatter on that page

https://vitepress.dev/reference/frontmatter-config#outline

**Reply by linusang-spark** [reply]
~~thanks but I also mean the main content will also span the width to take up the space by the table of contents. is it possible?~~ sorry i found the config `aside: false`
